pkg/cluster-runtime/installer.go

```diff
@@ -188,6 +188,19 @@ func (i *Installer) Install() error {
 		return err
 	}
 
+	for _, ip := range all {
```

Comment:
what if the new joined node need to do some  taint issues ? if so ,we also  need to do it in join